### PR TITLE
adding azul openjdk9 to all images

### PR DESCRIPTION
**Pull request by nlopezgi**
* For now we just add the packages to the images and file existence +
version tests

### CODE REVIEW

File: container/debian8/builders/rbe-debian8/BUILD [hunk 1]
```diff
@@ -64,6 +64,7 @@ toolchain_container(
         "//container/debian8/layers/clang:clang-ltl",
         "//container/debian8/layers/go:go-ltl",
         "//container/debian8/layers/java:java-ltl",
+        "//container/ubuntu16_04/layers/java:java9-ltl",
```
Comment (xingao267) [inline container/debian8/builders/rbe-debian8/BUILD]: "//container/debian8/layers/java:java9-ltl",

File: WORKSPACE [hunk 2]
```diff
@@ -254,6 +254,19 @@ http_file(
...
 )
 
+load(
+    "//third_party/openjdk:revision.bzl",
+    "JDK_VERSION",
+    "OPENJDK_SHA256",
+)
+
+# Axul JDK (from Bazel's OpenJDK Mirror)
+http_file(
```
Comment (xingao267) [inline WORKSPACE]: Any licensing we need to add?

Comment (nlopezgi) [reply WORKSPACE]: great question. I'll double check.

File: container/common/java.yaml [hunk 3]
```diff
@@ -15,6 +20,9 @@ fileExistenceTests:
 - name: 'OpenJDK'
   path: '/usr/lib/jvm/java-8-openjdk-amd64'
   shouldExist: true
+- name: 'OpenJDK9'
+  path: '/usr/lib/jvm/zulu9.0.7.1-jdk9.0.7-linux_x64'
```
[the remark this answers is not in the document]
Comment (xingao267) [reply container/common/java.yaml]: Missed this one?

[the remark this answers is not in the document]
Comment (nlopezgi) [reply container/common/java.yaml]: done

File: container/common/java.yaml [hunk 4]
```diff
@@ -6,6 +6,11 @@ commandTests:
   args: ['-version']
   # java outputs to stderr.
   expectedError: ["openjdk version \"1.8.*"]
+- name: 'java9-version'
+  command: '/usr/lib/jvm/zulu9.0.7.1-jdk9.0.7-linux_x64/bin/java'
```
[the remark this answers is not in the document]
Comment (nlopezgi) [reply container/common/java.yaml]: done

**Comment by nlopezgi** [comment]
Switched to using java 10

**Pull request merged**